packages/api-db-mongodb/src/db/peer.ts

```diff
@@ -27,7 +27,9 @@ export class MongoDBPeerAdapter implements DBPeerAdapter {
     if (!value) {
       return {
         name: '',
-        themeColor: '#000000'
+        themeColor: '#000000',
+        callToActionURL: '',
+        callToActionText: []
```

Comment:
Adding an empty array to a RichText makes it unwritable in the editor. Checkout `createDefaultValue()` in `packages/editor/src/client/blocks/richTextBlock.tsx`.

Reply:
@roosnic1 I fixed it from the editor, do you mean to have the default value here?